includes/media_bynder.formatters.inc

```diff
@@ -10,7 +10,8 @@
  * Implements hook_file_formatter_info().
  * $info: Array of information on file formatters exposed by hook_file_formatter_info() implementations.
  */
-function media_bynder_file_formatter_info_alter(&$info) {
+function media_bynder_file_formatter_info_alter(&$info)
```

Comment:
OOP PHP vs Drupal 7, the camel case ones are normally PHP classes implemented in external libraries and adhering to proper PHP coding standards. Underscore ones are Drupal 7 standards for function names